packages/itwinui-css/src/color-picker/color-picker.scss

```diff
@@ -228,13 +228,20 @@ $iui-hover-box-shadow: 0 0 0 var(--iui-size-3xs) var(--iui-color-border);
   border-radius: 50%;
   transform: translate(calc(0px - var(--iui-size-xs)), calc(0px - var(--iui-size-xs)));
   cursor: crosshair;
-  box-shadow: rgba(0, 0, 0, var(--iui-opacity-1)) 0 0 0 calc(var(--iui-size-3xs) - 1px),
-    inset 0 0 0 calc(var(--iui-size-3xs) - 1px) rgba(0, 0, 0, var(--iui-opacity-6));
   background-color: var(--iui-color-picker-selected-color);
 
-  &:hover {
-    box-shadow: $iui-hover-box-shadow;
+  &:focus-visible {
+    outline-offset: 3px;
```

Comment:
Can you think of a similar solution for the `:focus` outline?  It can also get lost depending on the background color.
![Screenshot 2023-08-15 at 11 02 43 AM](https://github.com/iTwin/iTwinUI/assets/849817/c6637dd4-2005-4d31-906f-8e0a6658f8d1)

Reply:
i don't want to make it too complicated, but one option is to use a two-tone focus ring
https://www.w3.org/WAI/WCAG21/Techniques/css/C40

Reply:
added secondary box-shadow, it should help a bit. can you test it out on your end?